workspaces/adoption-insights/plugins/adoption-insights/src/translations/fr.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTranslationMessages } from '@backstage/core-plugin-api/alpha';
+import { adoptionInsightsTranslationRef } from './ref';
+
+/**
+ * French translation for Adoption Insights.
+ * @public
+ */
+const adoptionInsightsTranslationFr = createTranslationMessages({
+  ref: adoptionInsightsTranslationRef,
+  messages: {
+    'page.title': "Insights d'Adoption",
+    'header.title': "Insights d'Adoption",
+    'header.dateRange.today': "Aujourd'hui",
+    'header.dateRange.lastWeek': 'La semaine dernière',
+    'header.dateRange.lastMonth': 'Le mois dernier',
+    'header.dateRange.last28Days': 'Les 28 derniers jours',
+    'header.dateRange.lastYear': "L'année dernière",
+    'header.dateRange.dateRange': 'Plage de dates...',
+    'header.dateRange.cancel': 'Annuler',
+    'header.dateRange.ok': 'OK',
+    'header.dateRange.defaultLabel': 'Les 28 derniers jours',
+    'activeUsers.title': 'Utilisateurs Actifs',
+    'activeUsers.averagePrefix': 'En moyenne',
+    'activeUsers.averageText': '{{count}} utilisateurs actifs par {{period}}',
+    'activeUsers.averageSuffix': 'étaient actifs au cours de cette période.',
+    'activeUsers.hour': 'heure',
+    'activeUsers.day': 'jour',
+    'activeUsers.legend.newUsers': 'Nouveaux utilisateurs',
+    'activeUsers.legend.returningUsers': 'Utilisateurs récurrents',
+    'templates.title': 'Top modèles',
+    'templates.topNTitle': 'Top {{count}} modèles',
+    'catalogEntities.title': 'Top entités du catalogue',
+    'plugins.title': 'Top plugins',
+    'plugins.topNTitle': 'Top {{count}} plugins',
+    'techDocs.title': 'Top TechDocs',
+    'techDocs.topNTitle': 'Top {{count}} TechDocs',
+    'searches.title': 'Top recherches',
+    'searches.totalCount': '{{count}} recherches',
+    'searches.averagePrefix': 'En moyenne',
+    'searches.averageText': '{{count}} recherches par {{period}}',
+    'searches.averageSuffix': 'ont été effectuées durant cette période.',
+    'searches.hour': 'heure',
+    'searches.day': 'jour',
+    'users.title': "Nombre total d'utilisateurs",
+    'users.haveLoggedIn': 'se sont connectés',
+    'users.loggedInUsers': 'Utilisateurs connectés',
+    'users.licensed': 'Licenciés',
+    'users.ofTotal': 'sur {{total}}',
+    'users.tooltip':
+      "Définissez le nombre d'utilisateurs sous licence dans app-config.yaml",
```

Comment:
```suggestion
      "Définissez le nombre d'utilisateurs sous licence dans le fichier de configuration app-config.yaml",
```

Might be clearer to state that this is a config file.